package.json

```diff
@@ -38,7 +38,9 @@
   },
   "dependencies": {
     "acl": "~0.4.10",
+    "amazon-s3-uri": "0.0.3",
```

Comment:
This library is rather new, and doesn't have many stars. However, the implementation looks fine and isn't too complicated. We'll probably have to keep an eye on this library though.

Reply:
Even we follow the library updates there are few minor changes that need to be update to it. As i am a new bee to this need support from others to get into and complicated issues like path, switching, controllers, pre & post updates need to be checked very carefully.
Update me on this if any wrong.